apis_core/apis_entities/models.py

```diff
@@ -274,6 +274,26 @@ def get_duplicate_url(self):
             kwargs={"entity": entity, "pk": self.id},
         )
 
+    def merge_fields(self, other):
+        """
+        This method is intended to be extended by subclasses.
+        It is called by the `merge_with` method and should transfer data from `other` to `self`
+        """
+        # Inherited from RootObject
+        if self.name != other.name:
+            self.name = ", ".join([self.name, other.name])
+        # Boolean values
+        self.review = self.review and other.review
+        self.status = self.status and other.status
+        self.published = self.published and other.published
+        # date strings: we use the other date only if the self date is missing
+        self.start_date_written = self.start_date_written or other.start_date_written
+        self.end_date_written = self.end_date_written or other.end_date_written
+        # we simply concatenate strings
+        self.references = "\n".join(filter(None, [self.references, other.references]))
```

Comment:
see comment above. should be clear which comments stem from an object that was merged in the actual one (put in brackets or explicitly mentioned ... eg `references from merged objects:`